### DISCUSSION TITLE
Bear formula

### DISCUSSION THREAD

**Discussion by rizsotto** [Homebrew/discussions#2178] [Writing Formulae/Casks] [answered]
Hey, I'm the author of Bear. Thanks for making a package from my project!

I've noticed that [the formula to build the package](https://github.com/Homebrew/linuxbrew-core/blob/HEAD/Formula/bear.rb) is not optimal.

- it states that `sqlite` is a dependency, which is not the case.
- it states that `python` is a dependency, which is not the case.
  - It might be a build dependency if you are running the functional tests, but that's not happening.
- it does `rewrite_shebang` on the file `bin/bear` which is not necessary, since it is not a Python script.

These issues probably not breaking the functionality. But can improve/simplify the package build and installation. (cc: @chenrui333)

**Comment by carlocab** [comment, marked as answer]
Hi @rizsotto, I've opened Homebrew/homebrew-core#85700 to remove the unneeded dependencies.

In the future, please open a pull request for any changes you'd like to be implemented. Thanks!

**Reply by rizsotto** [reply]
Thanks @carlocab for your help!

**Reply by carlocab** [reply]
No problem. Thanks for letting us know!
